packs/caddy/variables.hcl

```diff
@@ -0,0 +1,93 @@
+variable "job_name" {
+  description = "The name to use as the job name which overrides using the pack name"
+  type        = string
+  // If "", the pack name will be used
+  default = ""
+}
+
+variable "datacenters" {
+  description = "A list of datacenters in the region which are eligible for task placement"
+  type        = list(string)
+  default     = ["dc1"]
+}
+
+variable "region" {
+  description = "The region where the job should be placed"
+  type        = string
+  default     = "global"
+}
+
+variable "namespace" {
+  description = "The namespace where the job should be placed."
+  type        = string
+  default     = "default"
+}
+
+variable "consul_service_name" {
+  description = "The consul service you wish to load balance"
+  type        = string
+  default     = "webapp"
+}
```

Comment:
It doesn't seem this variable is used; can it be removed or does the template need updating?